imageio/plugins/ffmpeg.py

```diff
@@ -112,45 +114,64 @@ class FfmpegFormat(Format):
         The number of frames per second. Default 10.
     codec : str
         the video codec to use. Default 'libx264', which represents the
-        widely available mpeg4.
+        widely available mpeg4. Except when saving .wmv files, then the
+        defaults is 'msmpeg4' which is more commonly supported for windows
+    quality : float
+        Video output quality. Default is 5. Uses variable bit rate. Highest
+        quality is 10, lowest is 0. Set to None to prevent variable bitrate
+        flags to FFMPEG so you can manually specify them using ffmpeg_params
+        instead.
     bitrate : int
-        A measure for quality. Default 400000
+        Set a constant bitrate for the video encoding. By default 'quality'
+        is used instead.
```

Comment:
Maybe be a bit more precise here:
* `quality : float | None`
* `bitrate : int | None` and say that the default is None
* Mention that if bitrate is given, it overrides quality.